Examples/SnapinsSDKExample/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimary">#3F51B5</color>
```

Comment:
I think I would add all salesforce brandable strings in this xml.
Then I would do something like:
<color name="colorPrimary">@color/salesforce_brand_primary</color>
or 
<color name="salesforce_brand_primary">@color/colorPrimary</color>
That way the developer can easily see how to theme snapins to work with their app and can easily test branding changes.